src/components/wallets/WalletTransactionList.tsx

```diff
@@ -46,149 +50,162 @@ interface WalletTransactionListProps {
   wallet: WalletInfosForTransactionsFragment
```

Comment:
You're just using wallet.id, so you can just pass the wallet id and remove the fragment (that is not really usefull here since status and currency are not used)